garage.toml

```diff
@@ -0,0 +1,29 @@
+metadata_dir = "/var/lib/garage/meta"
+data_dir = "/var/lib/garage/data"
+db_engine = "sqlite"
```

Comment:
We should use `lmdb` instead. See https://garagehq.deuxfleurs.fr/documentation/reference-manual/configuration/#db_engine

Reply:
Just read that page, I think we should stick with `sqlite` here due to the following issues in `lmdb`:

>Several users have reported corrupted LMDB database files after an unclean shutdown (e.g. a power outage). This situation can generally be recovered from if your cluster is geo-replicated (by rebuilding your metadata db from other nodes), or if you have saved regular snapshots at the filesystem level.

We will not have any snapshots or replication so in case of an unclean shutdown and DB corruption, it would be a nightmare (or impossible) to repair the storage.

>Keys in LMDB are limited to 511 bytes. This limit translates to limits on object keys in S3 and sort keys in K2V that are limted to 479 bytes.

Don't know how long our object keys are but I don't want to take any chances.

Sounds like `sqlite` should be good enough for our single-node, (hopefully) smaller scale deployment.

WDYT?